### PR TITLE
minor cleanup to the cleanup pr

### PR DESCRIPTION
**Pull request by iamigo**

### CODE REVIEW

File: src/sampleStore/sampleCleanup/cleanup.js [hunk 1]
```diff
@@ -34,9 +33,11 @@ module.exports = (redis) => new Promise((resolve, reject) => {
   return redis.smembers(samsto.key.samples)
   .then((s) => {
     samples = s;
-    debug('Check for all samples that are present in redis or not');
+    debug('Checking whether each member of the set has a corresponding ' +
+      `"${samsto.pfx.sample}[SAMPLE_NAME]" hash`);
     const commands = s.map(sample => ['exists', sample]);
 
+    // TODO unnest these nested promises
```
Comment (iamigo) [inline src/sampleStore/sampleCleanup/cleanup.js]: @harshkothari410

File: src/sampleStore/sampleCleanup/cleanup.js [hunk 2]
```diff
@@ -82,8 +83,12 @@ module.exports = (redis) => new Promise((resolve, reject) => {
     });
 
     stream.on('end', () => {
-      debug('End of scanning data');
-      debug('Removed samples list: %o', deletedSample);
+      debug(`Completed scan for "${samsto.pfx.sample}*" keys`);
+      console.log('===================== Deleted Samples ====================');
+      Array.from(new Set(deletedSample)).map(console.log);
+
+      // TODO print out the sample keys which were deleted from master list
```
Comment (iamigo) [inline src/sampleStore/sampleCleanup/cleanup.js]: @harshkothari410

File: src/sampleStore/sampleCleanup/previewCleanup.js [hunk 3]
```diff
@@ -32,13 +31,15 @@ let deletedSample = [];
...
-    samples = s;
-    debug('Check for all samples that are present in redis or not');
-    const commands = s.map(sample => ['exists', sample]);
+  .then((members) => {
+    samples = members;
+    debug('Checking whether each member of the set has a corresponding ' +
+      `"${samsto.pfx.sample}[SAMPLE_NAME]" hash`);
+    const commands = members.map(sample => ['exists', sample]);
 
+    // TODO unnest nested promise
```
Comment (iamigo) [inline src/sampleStore/sampleCleanup/previewCleanup.js]: @harshkothari410

File: src/sampleStore/sampleCleanup/cleanup.js (hunk 1 above)
Comment (harshkothari410) [reply src/sampleStore/sampleCleanup/cleanup.js]: Yes we can do this. True.

**Pull request merged**